### ISSUE TITLE
How do you change the text that is displayed after using the /send command?

### ISSUE THREAD

**Issue by c0wg0d** [PaperMC/Waterfall#477]
In a recent build there is a ton of text after issuing the /send command.  How do we get rid of all this so it just says "Sent <player> to <server>"?

**Comment by ghost** [comment]
In the bungeecord jar there is a messages.txt file or something like that

**Comment by c0wg0d** [comment]
Yes, I see there is a messages.properties file inside the jar, but I feel there should be a way for us to change the messages in a configuration file.

**Comment by electronicboy** [comment]
Those messages from /send are not configurable, no idea if there is an upstream issue on that
as for the messages file, just extract the file out of the jar and throw it in the same folder as you run the server from, no need to move anything to a config file, modifying that file outside of the jar has been supported for years

**Comment by LuminousLoreBloom** [comment]
If md_5 goes ahead with approving it; this PR I posted last week should support your needs
https://github.com/SpigotMC/BungeeCord/pull/3015

```
command_send_no_players=\u00a7cNo players selected to send
command_send_attempting=\u00a72Attempting to send {0} to {1}
command_send_delimiter=\u00a77, 
command_send_results=\u00a7aSend Results: 
command_send_counter=\u00a76({0})
command_send_result_title=\u00a7b{0}
command_send_hover_name_prefix=\u00a7e
command_send_hover_delimiter=\u00a77, 
```

**Comment by c0wg0d** [comment]
> If md_5 goes ahead with approving it; this PR I posted last week should support your needs
> [SpigotMC/BungeeCord#3015](https://github.com/SpigotMC/BungeeCord/pull/3015)

That looks great!  We only ever send one player at a time, so all that extra info is just totally unnecessary and clogs up the chat.  All I want is to see "_player_ has been sent to _server_" or something like that.
